packages/core/src/plugins/rollup/dynamic-import.ts

```diff
@@ -0,0 +1,10 @@
+import type { Plugin } from 'rollup';
+
+import type { Format } from '../../types';
+
+export const dynamicImport = (format: Format): Plugin => ({
+  name: 'crackle:resolve-dynamic-imports',
+  renderDynamicImport() {
+    if (format === 'cjs') return { left: 'import(', right: ')' };
+  },
+});
```

Comment:
Is this adding support for dynamic imports to CJS files?

Reply:
It's been a while (and Rollup 3 came out in the mean time), but this [might not be needed any more](https://rollupjs.org/migration/#dynamic-import-in-commonjs-output).